backend/src/test/java/com/daedan/festabook/place/service/PlaceImageServiceTest.java

```diff
@@ -84,14 +88,15 @@ class addPlaceImage {
         void 예외_존재하지_않는_플레이스() {
             // given
             Long invalidPlaceId = 1L;
+            Long festivalId = 1L;
 
             given(placeJpaRepository.findById(invalidPlaceId))
                     .willReturn(Optional.empty());
 
             PlaceImageRequest placeImageRequest = PlaceImageRequestFixture.create();
```

Comment:
아래 메서드에서 인스턴스 이름을 request로 변경했네요.
이곳도 동일하게 변경해주세요.

```suggestion
            PlaceImageRequest request = PlaceImageRequestFixture.create();
```